pyproject.toml

```diff
@@ -0,0 +1,135 @@
+license = { file = "LICENSE" }
+keywords = ["Django", "Django-CMS", "Open edX"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: Django",
+    "Framework :: Django :: 2.2",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7"
+]
+dependencies = [
+    "arrow",
+    "Django<4",
+    "djangocms-file",
+    "djangocms-googlemap",
+    "djangocms-link",
+    "djangocms-picture",
+    "djangocms-text-ckeditor",
+    "djangocms-video",
+    "djangorestframework",
+    "django-autocomplete-light==3.9.7",
+    "django-cms>=3.11.0",
+    "django-parler>=2.3",
+    "django-redis>=4.11.0",
+    "django-treebeard",
+    "dj-pagination",
+    "easy_thumbnails[svg]>=2.8",
+    "elasticsearch>=6.0.0,<7.0.0",
+    "exrex==0.11.0",
+    "importlib-metadata",
+    "oauthlib==3.2.2",
+    "requests==2.31.0",
+]
+
+[project.urls]
+"Bug Tracker" = "https://github.com/openfun/richie/issues/new"
+"Changelog" = "https://github.com/openfun/richie/blob/master/CHANGELOG.md"
+"Homepage" = "https://github.com/openfun/richie"
+"Repository" = "https://github.com/openfun/richie"
+"Documentation" = "https://richie.education/"
+"Demonstration website" = "https://demo.richie.education/"
+
+[tool.setuptools]
+packages = { find = { where = ["src"], exclude = ["tests"] } }
+zip-safe = true
+
+[tool.distutils.bdist_wheel]
+universal = true
+
+[project.optional-dependencies]
+dev = [
+    "bandit==1.7.5",
+    "black==23.11.0",
+    "pyRdfa3==3.5.3",
+    "cssselect==1.2.0",
+    "factory-boy==3.3.0",
+    "flake8==6.1.0",
+    "flake8-pyproject==1.2.3",
+    "htmlmin==0.1.12",
+    "ipdb==0.13.13",
+    "ipython==8.17.2",
+    "isort==5.12.0",
+    "lxml==4.9.3",
+    "msgpack==1.0.7",
+    "mysqlclient==2.2.0",
+    "pylint==3.0.2",
+    "pylint-django==2.5.5",
+    "pytest==7.4.3",
+    "pytest-cov==4.1.0",
+    "pytest-django==4.7.0",
+    "responses==0.24.1",
+]
+ci = [
+    "twine==4.0.2",
+]
+sandbox = [
+    "django-configurations==2.5",
+    "dockerflow==2022.8.0",
+    "factory-boy==3.3.0",
+    "gunicorn==21.2.0",
+    "psycopg2-binary==2.9.9",
+    "sentry-sdk==1.35.0",
+]
+
+[tool.flake8]
+max-line-length = 99
+exclude = [
+    ".git",
+    ".venv",
+    "build",
+    "venv",
+    "__pycache__",
+    "node_modules",
+    "*/migrations/*",
+]
+
+[tool.isort]
+profile = "black"
+known_django = "django"
+known_richie = "richie"
+include_trailing_comma = true
+line_length = 88
+multi_line_output = 3
+sections = ["FUTURE","STDLIB","DJANGO","THIRDPARTY","RICHIE","FIRSTPARTY","LOCALFOLDER"]
+skip_glob = "src/frontend/node_modules/**/*,venv"
+
+[tool.pytest.ini_options]
+addopts = "-v --cov-report term-missing"
+python_files = [
+    "test_*.py",
+    "tests.py"
+]
+testpaths = [
+    "tests",
+]
+filterwarnings = [
+    "ignore:::(?!(tests|richie))",
+]
+
+[tool.setuptools_scm]
```

Comment:
Is it mandatory to declare an empty table ?

Reply:
Yes